app/controllers/concerns/new_inprocess_controller.rb

```diff
@@ -13,6 +13,8 @@ def index
                                               TransactionSearch::OrderStatusSearcher,
                                               TransactionSearch::DateRangeSearcher).search(order_details, @search_form)
     @order_details = @search.order_details.includes(:order_status).joins_assigned_users.reorder(sort_clause)
+    @date_range_field = @search_form.date_params[:field]
+    @use_estimated_cost_header = true
 
     respond_to do |format|
       format.html { @order_details = @order_details.paginate(page: params[:page]) }
```

Comment:
I think that the html view's header label(s) (2 controllers) may need to be adjusted too.

Reply:
The views use a generic label of "Price".  The costs listed are green for actual, red for estimated, or display "Unassigned" if unassigned.  I think it's worth adding a comment but we don't need to change the header text.